# auth/exeptions/__inity__.py

from .invalidCredentialsException import InvalidCredentialsException
from .invalidAuthCodeException import InvalidAuthCodeException
